lib/backend/syncersv1/bgpsyncer/bgpsyncer.go

```diff
@@ -57,9 +57,11 @@ func New(client api.Client, callbacks api.SyncerCallbacks, node string, watchAll
 			ListInterface:   model.ResourceListOptions{Kind: apiv3.KindBGPPeer},
 			UpdateProcessor: updateprocessors.NewBGPPeerUpdateProcessor(),
 		},
-		{
+	}
+	if node != "" {
```

Comment:
Maybe worth adding a comment here that we only expect node to be blank for the RR and in that case we don't need to monitor the BlockAffinities